src/compiler/parser/index.js

```diff
@@ -38,7 +38,7 @@ const modifierRE = /\.[^.\]]+(?=[^\]]*$)/g
 const slotRE = /^v-slot(:|$)|^#/
 
 const lineBreakRE = /[\r\n]/
-const whitespaceRE = /\s+/g
+const whitespaceRE = /[ \t\r\n]+/g
```

Comment:
Maybe we should use *space characters* defined in HTML spec [here](https://www.w3.org/TR/html52/infrastructure.html#space-characters)?

Reply:
Yeah, thats a good idea. The only possible char missing from the regex at this point would be `U+000C FORM FEED (FF)`